src/components/dialogs/network-modifications/line-attach-to-voltage-level/line-attach-to-voltage-level-form.jsx

```diff
@@ -120,6 +131,18 @@ const LineAttachToVoltageLevelForm = ({
             <Grid container spacing={2}>
                 <GridItem>{attachmentPointIdField}</GridItem>
                 <GridItem>{attachmentPointNameField}</GridItem>
+                <GridItem>
+                    {
+                        <Button
+                            onClick={openAttachmentPointDialog}
+                            startIcon={isVoltageLevelEdit ? <EditIcon /> : <AddIcon />}
```

Comment:
You never edit a voltage level so isVoltageLevelEdit is false and this will always be `<AddIcon />`.

Having a `<EditIcon />` would be nice though. But I think it would work better like this : 
```suggestion
                            startIcon={attachmentPoint != null ? <EditIcon /> : <AddIcon />}
```